ember-app/app/controllers/issue/create.js

```diff
@@ -18,6 +18,10 @@ var IssuesCreateController = Ember.Controller.extend({
   }.observes('selectedRepo'),
   otherLabels: Ember.computed.alias("selectedRepo.data.other_labels"),
   assignees: Ember.computed.alias("selectedRepo.data.assignees"),
+  //Ugly Hack for backwards Enterprise Compatibiltiy
+  assigneesEnabled: function(){
+    return this.get("selectedRepo.board.issues").isAny("assignees");
```

Comment:
Right. It's efficient when `assignees` is expected to be found, but will scan every issue looking for `assignees` when it's not found.